client/java-armeria-legacy/src/main/java/com/linecorp/centraldogma/client/armeria/legacy/LegacyCentralDogma.java

```diff
@@ -230,6 +233,38 @@
         });
     }
 
+    @Override
+    public CompletableFuture<AggregatedEntry<?>> aggregateEntries(String projectName, String repositoryName,
+                                                                  Revision revision,
+                                                                  List<PathAndOptional> paths,
+                                                                  List<String> expressions) {
```

Comment:
How about introducing `AggregationQuery<T>` so both paths and expressions are presented in a single type, just like we do in `getFile()` and `Query<T>`? We could also add a shortcut method similar to `getFile(projectName, repositoryName, revision, path)`.